.gitignore

```diff
@@ -24,3 +24,6 @@ testbin/*
 *.swp
 *.swo
 *~
+
+# vendor files
+vendor
```

Comment:
This will directly overwrite the L19 ? 
Do we still need L19?

Reply:
Agree
```bash
# Kubernetes Generated files - skip generated files, except for vendored files

!vendor/**/zz_generated.*
```
if we are going to ignore the entire folder, let's have just one line for it

Reply:
I commented out the `!vendor/**/zz_generated.*` line. Once it's decided to include vendor in the repo, we can uncomment that line.